CONTRIBUTING.md

```diff
@@ -25,8 +25,16 @@ guidance below.
 ## Changing the code-base
 
 Generally speaking, you should fork this repository, make changes in your
-own fork, and then submit a pull-request. All new code should have associated unit
```

Comment:
I don't think we've had any standardization around that though, have we? 

I mean, clearly the CONTRIBUTING file hasn't reflected that practice :)